src/test/java/org/magicdgs/readtools/utils/trimming/TrimAndFilterPipelineUnitTest.java

```diff
@@ -544,7 +546,7 @@ public void testParsingAndPipelineFromPlugin(final ArgumentsBuilder arguments,
             throws Exception {
         final CommandLineParser clp = new CommandLineArgumentParser(new Object(), Arrays.asList(
                 new GATKReadFilterPluginDescriptor(defaultFilters),
-                new TrimmerPluginDescriptor(defaultTrimmers)),
+                new TrimmerPluginDescriptor(new TrimReadsTrimmerPluginArgumentCollection(), defaultTrimmers)),
```

Comment:
This test is for integration of the plugin and the `TrimReads` argument collection. It should be moved out of here.

Reply:
Moved to TrimReadsTrimmerPluginArgumentCollectionTest